src/gc/gcee.cpp

```diff
@@ -12,19 +12,16 @@
 
 #ifndef DACCESS_COMPILE
 
-#ifdef ENABLE_PERF_COUNTERS
-PERF_COUNTER_TIMER_PRECISION g_TotalTimeInGC = 0;
-PERF_COUNTER_TIMER_PRECISION g_TotalTimeSinceLastGCEnd = 0;
-#endif
+GCCounterData g_GCCounterData;
+
+uint64_t g_TotalTimeInGC = 0;
+uint64_t g_TotalTimeSinceLastGCEnd = 0;
 
-#if defined(ENABLE_PERF_COUNTERS) || defined(FEATURE_EVENT_TRACE)
 size_t g_GenerationSizes[NUMBERGENERATIONS];
 size_t g_GenerationPromotedSizes[NUMBERGENERATIONS];
-#endif // ENABLE_PERF_COUNTERS || FEATURE_EVENT_TRACE
 
 void GCHeap::UpdatePreGCCounters()
 {
-#if defined(ENABLE_PERF_COUNTERS)
```

Comment:
I don't understand the code organization here - you seem to have deleted some code related to the original PERF_COUNTERS support but not all of them. eg you moved the related code in UpdatePreGCCounters but not in UpdatePostGCCounters. why is this? and it doesn't look like an effort to replace, eg, you still kept the timeInGC data in PERF_COUNTERS support.

Reply:
Removed in the latest commit